array_filter.json

```diff
@@ -0,0 +1,50 @@
+{
+    "id": "array_filter",
+    "summary": "Filter an array based on a logical expression",
+    "description": "Filters the array elements based on a logical expression so that afterwards an array is returned that only contains the values conforming to the expression.",
+    "categories": [
+        "arrays",
+        "filter"
+    ],
+    "parameter_order": [
+        "data",
+        "expression"
```

Comment:
in #128 , the usage of "expression" was replaced by "condition" (for clarity reasons I guess). Maybe also do that here?

Reply:
Good catch, fixed.